native/cocos/application/ApplicationManager.h

```diff
@@ -73,6 +73,7 @@ class CC_DLL ApplicationManager {
 #define CC_CURRENT_APPLICATION_SAFE()   CC_APPLICATION_MANAGER()->getCurrentAppSafe()
 #define CC_CURRENT_ENGINE()             CC_CURRENT_APPLICATION_SAFE()->getEngine()
 #define CC_GET_PLATFORM_INTERFACE(intf) CC_CURRENT_ENGINE()->getInterface<intf>()
+#define CC_GET_XR_INTERFACE() BasePlatform::getPlatform()->getInterface<IXRInterface>()
```

Comment:
What will happen if it is not xr platform?

Reply:
nullptr
It is necessary to judge the null pointer when using